src/bytecode/mod.rs

```diff
@@ -0,0 +1,51 @@
+pub mod instructions;
+
+use std::vec::Vec;
+use std::fmt;
+use self::instructions::Instr;
+
+/// A simpler representation of Lua
+pub struct LuaBytecode {
+    block: Vec<Instr>,
+    ids: u64
```

Comment:
What does this do?

Ah, I see in `get_new_var` what it's for. Maybe this should be called something like `reg_counter`?

Reply:
That sounds better!